src/components/huskelapp/HuskelappModal.tsx

```diff
@@ -106,32 +89,32 @@ export const HuskelappModal = ({ isOpen, toggleOpen }: HuskelappModalProps) => {
             onChange={oppdaterTekst}
           />
         )}
-        <ButtonRow topPadding={PaddingSize.SM}>
-          <Button
-            variant="primary"
-            onClick={handleOppdaterHuskelapp}
-            loading={oppdaterHuskelapp.isLoading}
-            disabled={isLoading}
-          >
-            {texts.save}
-          </Button>
-          <Button variant="secondary" onClick={() => toggleOpen(false)}>
-            {texts.close}
-          </Button>
-          {huskelapp && (
-            <Tooltip content={texts.removeTooltip}>
-              <RemoveButton
-                icon={<TrashIcon aria-hidden />}
-                variant="danger"
-                onClick={() => handleRemoveHuskelapp(huskelapp.uuid)}
-                loading={removeHuskelapp.isLoading}
-              >
-                {texts.remove}
-              </RemoveButton>
-            </Tooltip>
-          )}
-        </ButtonRow>
       </ModalContent>
-    </StyledModal>
+      <Modal.Footer>
+        <Button
+          variant="primary"
+          onClick={handleOppdaterHuskelapp}
+          loading={oppdaterHuskelapp.isLoading}
+          disabled={isLoading}
```

Comment:
Er det redundant å ha begge her? Virker som den er disabled når `loading == true`😊
```suggestion
          loading={oppdaterHuskelapp.isLoading}
```

Reply:
Jeg har ikke endret noe på dette i denne sammenhengen, men `isLoading` er `true` når vi gjør GET for å hente huskelapp, mens `oppdaterHuskelapp.isLoading` er `true` når man gjør POST, så den endringen vil i så fall føre til en endring i oppførselen her :thinking:

Reply:
Woops. Jeg misforstod hvor denne👇 startet og sluttet. Nevermind 😊
```
{isLoading && <HuskelappSkeleton />}
        {isSuccess && (…      
```